WordPress/Classes/Services/BlogService.m

```diff
@@ -108,12 +124,15 @@ - (Blog *)lastUsedBlog
         return nil;
     }
 
-    NSFetchRequest *fetchRequest = [NSFetchRequest fetchRequestWithEntityName:@"Blog"];
-    NSPredicate *predicate = [NSPredicate predicateWithFormat:@"visible = YES AND url = %@", url];
+    NSFetchRequest *fetchRequest = [NSFetchRequest fetchRequestWithEntityName:BlogEntity];
+    NSPredicate *predicate = [NSPredicate predicateWithFormat:@"%@ AND url = %@", @"visible = YES", url];
```

Comment:
This isn't necessarily a bad thing, but I find this to be less legible than what we had before.  Was there any particular reason to add this change?